### ISSUE TITLE
Privacy policy dialog doesn’t have either aria-label or aria-labelledby

### ISSUE THREAD

**Issue by sbates-idrc** [codelearncreate/c2lc-coding-environment#471]
Somehow VoiceOver on MacOS reads the title of the modal when the privacy modal opens up, but I realized that’s not the case for VoiceOver on iPad, or NVDA on windows. Our custom modal component has optional properties for aria-label and aria-labelledby, so we should set one of them to fix this issue.

Original Jira issue: https://fluidproject.atlassian.net/browse/C2LC-634
